src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -902,16 +901,21 @@ export default class Aircraft {
             expedite: expedite
         });
 
-        let isExpeditingString = '';
-        if (expedite) {
-            isExpeditingString = 'and expedite';
-        }
+        const newAltitude = this.fms.altitudeForCurrentWaypoint();
+        const instruction = radio_trend('altitude', this.altitude, newAltitude);
+        const readback_text = `${instruction} ${newAltitude}`;
+        const readback_verbal = `${instruction} ${radio_altitude(newAltitude)}`;
 
         const readback = {
-            log: `${radioTrendAltitude} ${this.fms.altitudeForCurrentWaypoint()} ${isExpeditingString}`,
-            say: `${radioTrendAltitude} ${currentWaypointRadioAltitude} ${isExpeditingString}`
+            log: `${readback_text}`,
+            say: `${readback_verbal}`
         };
 
+        if (expedite) {
+            readback.log = `${readback_text} and expedite`;
+            readback.say = `${readback_verbal} and expedite`;
```

Comment:
here, though, you _do_ need to use template strings. 

👍 this looks so much nicer.